src/test/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheckTest.java

```diff
@@ -378,6 +382,58 @@ public void testCountMatches() throws Exception {
         assertEquals("Unexpected matches count", 1, actual);
     }
 
+    /**
+     * Testing, that all elements in the constant NON_PRINTABLE_CHARS are sorted.
+     * This is very convenient for the sake of maintainability.
+     *
+     * @throws Exception when code tested throws some exception
+     */
+    @Test
+    public void testNonPrintableCharsAreSorted() throws Exception {
+        // Getting Field Value via Reflection, becaues the field is private
+        final Field field = TestUtil.getClassDeclaredField(
+                AvoidEscapedUnicodeCharactersCheck.class, "NON_PRINTABLE_CHARS");
+        field.setAccessible(true);
```

Comment:
Please look at `getClassDeclaredField` method. It is already specifying `setAccessible` so this line isn't needed.
Remove it.

Reply:
Done